src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/DesignTimeTargets/Microsoft.Managed.DesignTime.targets

```diff
@@ -147,36 +157,40 @@
       <Context>File</Context>
     </PropertyPageSchema>
 
-    <PropertyPageSchema Include="$(ManagedXamlResourcesDirectory)Compile.BrowseObject.xaml;">
-      <Context>BrowseObject</Context>
-    </PropertyPageSchema>
-
     <PropertyPageSchema Include="$(ManagedXamlNeutralResourcesDirectory)Content.xaml">
       <Context>File</Context>
     </PropertyPageSchema>
 
-    <PropertyPageSchema Include="$(ManagedXamlResourcesDirectory)Content.BrowseObject.xaml">
-      <Context>BrowseObject</Context>
-    </PropertyPageSchema>
-    
     <PropertyPageSchema Include="$(ManagedXamlNeutralResourcesDirectory)EmbeddedResource.xaml">
       <Context>File</Context>
     </PropertyPageSchema>
 
-    <PropertyPageSchema Include="$(ManagedXamlResourcesDirectory)EmbeddedResource.BrowseObject.xaml">
-      <Context>BrowseObject</Context>
+    <PropertyPageSchema Include="$(ManagedXamlNeutralResourcesDirectory)None.xaml">
```

Comment:
>ManagedXamlNeutralResourcesDirectory [](http://example.com/codeflow?start=35&length=36)

It appears most of the old files used to all be called *browseobject.xaml and have different folders. They now reside in the same folder with different names. However, I do not see these new files being added. Were they preexisting?

Reply:
Or am I confused because they order might have changed?

Reply:
There are no changes in directories.

Perhaps this sort is confusing. I'll revert this sort.

Reply:
you can leave it. It was just confused by the diff and thought you'd forgotten to add files.